src/async-data/actions.js

```diff
@@ -0,0 +1,34 @@
+export const actionTypes = ['REQUESTED', 'RETRY', 'SUCCEEDED', 'FAILED', 'REPLACE', 'RESET']
+
+export const createActionTypes = (actionTypes, prefix) =>
+	actionTypes.reduce((res, k) => ({ ...res, [k]: `${prefix}_${k}` }), {})
+
+export const createActions = prefix => {
+	const actions = createActionTypes(actionTypes, prefix)
+	return {
+		requested: params => ({
```

Comment:
Is params expected to be a shallow object or a deep object? In the later case spreading will not create an inmutable copy.